pkgs/development/libraries/libfprint-2-tod1-vfs0090/default.nix

```diff
@@ -38,5 +38,6 @@ stdenv.mkDerivation {
     license = licenses.lgpl21Plus;
     platforms = platforms.linux;
     maintainers = with maintainers; [ valodim ];
+    broken = true;
```

Comment:
@Valodim I need to update `libfprint-tod` but this breaks `libfprint-2-tod1-vfs0090`. Checking the GitLab, I see that `libfprint-2-tod1-vfs0090` is also several versions behind. Can you please update the package and test it? I don't have the hardware to do so.

Reply:
sadly, in practice the fingerprint reading was so slow that it fell out of use for me :(

I'll still try to do the update though, unless this causes conflicts with the `libfprint-tod` related to the below problem with goodix?